presto-docs/src/main/sphinx/prestissimo/prestissimo-features.rst

```diff
@@ -184,3 +184,184 @@ for cleanup. Only applicable when ``enable-old-task-cleanup`` is ``true``.
 Old task is defined as a PrestoTask which has not received heartbeat for at least
 ``old-task-cleanup-ms``, or is not running and has an end time more than
 ``old-task-cleanup-ms`` ago.
+
+
+Session Properties
+------------------
+
+Defines all system session properties supported by native worker to ensure
+that they are the source of truth and to differentiate them from Java based
+session properties.
+
+The following are the native session properties kept as the source of truth in prestissimo.
+
+``driver_cpu_time_slice_limit_ms``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``integer``
+* **Default value:** ``1000``
+
+Native Execution only. ``driver_cpu_time_slice_limit_ms`` defines the maximum CPU time,
+in milliseconds, that a driver thread is permitted to run before it must yield to other threads,
+facilitating fair CPU usage across multiple threads.
+A positive value enforces this limit, ensuring threads do not monopolize CPU resources.
+Setting this property to ``zero`` allows a thread to run indefinitely without yielding,
+which is not recommended in a shared environment as it can lead to resource contention.
+Negative values are considered invalid and are treated as a request to use the system default setting,
+which is ``1000`` ms in this case.
+
+``legacy_timestamp``
+^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``boolean``
+* **Default value:** ``true``
+
+Native Execution only. Use legacy TIME & TIMESTAMP semantics.
+Warning: this will be removed.
+
+``native_aggregation_spill_memory_threshold``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``integer``
+* **Default value:** ``0``
+
+Native Execution only. Specifies the maximum memory, in bytes,
+that a final aggregation operation can utilize before it starts spilling to disk.
+If set to ``0``, there is no limit, allowing the aggregation to consume unlimited memory resources,
+which may impact system performance.
+
+``native_debug_validate_output_from_operators``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``boolean``
+* **Default value:** ``false``
+
+If set to ``true``, then during the execution of tasks, the output vectors of every operator are validated for consistency.
+This is an expensive check and should only be used for debugging purposes.
```

Comment:
Consider moving this sentence as a Note below the other two sentences to emphasize this for the reader.

Reply:
updated